test/utils.py

```diff
@@ -0,0 +1,42 @@
+from shlex import split
+from subprocess import run
+
+
+def determine_scope(
+    fixture_name,
+    config
+) -> str:
```

Comment:
This function signature isn't very long so let's unwrap it

```suggestion
def determine_scope(fixture_name, config) -> str:
```